static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -143,11 +146,19 @@ function Task({task, completed, hidePanel}: TaskProps) {
     [task, organization, api]
   );
 
-  if (completed) {
+  if (status === 'complete' || status === 'skipped') {
     return (
       <TaskWrapper css={taskCompletedCss}>
         <strong>{task.title}</strong>
-        <IconCheckmark color="green300" isCircled />
+        {status === 'complete' ? (
+          <Tooltip title={t('Task completed ')} containerDisplayMode="flex">
+            <IconCheckmark color="green300" isCircled />
+          </Tooltip>
+        ) : (
+          <Tooltip title={t('Task skipped')} containerDisplayMode="flex">
+            <IconCheckmark color="gray300" isCircled />
+          </Tooltip>
+        )}
```

Comment:
@ale-cota what do you think? maybe it's too much to show in completed tasks, but I thought I'd make it consistent